File: web3rpc/rpc-specs/code-samples/python/eth/account/getBalance.py
from web3 import Web3
from web3py_ext import extend

host = "https://public-en-kairos.node.kaia.io"

address = "0x1cbd3b2770909d4e10f157cabc84c7264073c9ec"
blockNumberOrHash = "latest"

w3 = Web3(Web3.HTTPProvider(host))
eth_response = w3.eth.get_balance(address, blockNumberOrHash)

print(eth_response)

File: web3rpc/rpc-specs/code-samples/javascript/eth/account/getBalance.js
const { JsonRpcProvider } = require("@kaiachain/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-kairos.node.kaia.io");
  const address = "0x3111a0577f322e8fb54f78d9982a26ae7ca0f722";
  const blockNumberOrHashOrTag = "latest";

  provider.eth
    .getBalance(
      address,
      blockNumberOrHashOrTag,
      {},
      (err, data, response) => {},
    )
    .then((data) => {
      console.log(data);
    });
})();
